### ISSUE TITLE
two more issues

### ISSUE THREAD

**Issue by ifangcheng** [fucusy/Cross-View-Gait-Based-Human-Identification-with-Deep-CNNs#2]
many thx for the previous reply！

two more issues：

1. have you tested the model on the CASIA_B dataset？  can you achieve the 90% around accuracy？

2. have you observed the final performance difference between  using  nn.LogSoftMax and nn.SoftMax （since you used nn.LogSoftMax here）

**Comment by fucusy** [comment]
1. I have not tried, I guess you can achieve around 90%.
2. If you use nn.ClassNLLCriterion as loss function you need to add nn.LogSoftMax as your last layer. you can see more from  [nn.ClassNLLCriterion document here.](https://github.com/torch/nn/blob/master/doc/criterion.md#nn.ClassNLLCriterion), if you use nn.Softmax, I do not find suitable Criterion class as loss function, you may write the Criterion youself. Besides, combining Log and SoftMax  is for speed purpose.
